### PR TITLE
Avoid build storage for graphd image building. And try the local modu…

### PR DESCRIPTION
**Pull request by Shylock-Hg**
…les first.

### CODE REVIEW

File: package/CMakeLists.txt [hunk 1]
```diff
@@ -18,6 +18,8 @@ cmake_minimum_required(VERSION 3.5.0)
 # Set the project name
 project("Nebula Package")
 
+OPTION(ENABLE_PACKAGE_STORAGE "Whether package storage." ON)
```
Comment (yixinglu) [inline package/CMakeLists.txt:21]: Why not use `ENABLE_BUILD_STORAGE` to control the package process ?

Comment (Shylock-Hg) [reply package/CMakeLists.txt:21]: It's not the option of this camke file.

Comment (yixinglu) [reply package/CMakeLists.txt:21]: Could the CMakeList.txt in subdirectory not use global option ?

Comment (Shylock-Hg) [reply package/CMakeLists.txt:21]: They are different cmake building.This is just for package.

**Review by yixinglu — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**